frontend/src/features/leases/add/ReferenceSubForm.tsx

```diff
@@ -45,16 +45,6 @@ const ReferenceSubForm: React.FunctionComponent<IReferenceSubFormProps> = props
           />
         </Col>
       </Row>
-      <Row>
```

Comment:
the tfa and ps numbers are copied in their original form into the lease notes, also including them here is confusing and non-searchable